### DISCUSSION TITLE
sync without pushing to Git Repo

### DISCUSSION THREAD

**Discussion by IsiR19** [warmuuh/milkman#85] [Q&A] [answered]
Hi

I would like to find out is there a way to sync milkman with Git repo without doing a push to the repo?

**Comment by warmuuh** [comment, marked as answer]
what would be the usecase?  currently, the process is pretty much hardcoded (fetch, merge, push)

**Reply by IsiR19** [reply]
so the issue we have is, we have multiple people using milkman, but when guys are doing a sync it tends to break the bitbucket repo.

**Reply by IsiR19** [reply]
So the idea we were thinking of is having 1 guy keeping the repo up to date so have write access and the others just being able to pull

**Reply by warmuuh** [reply]
I have to admit, I have been chasing this repo-sync bug for ages but not successful. But your idea sounds like a little workaround, which might be easy to add. I will look into a read-only mode where changes can only be pulled but nothing can be pushed. Will create a ticket for this. Can't say though when I find time for it to implement

**Reply by warmuuh** [reply]
created https://github.com/warmuuh/milkman/issues/88
